### PR TITLE
NXP-28808: add retention pages translation

### PR DESCRIPTION
**Pull request by gcarlin**

### CODE REVIEW

File: i18n/messages.json [hunk 1]
```diff
@@ -1250,4 +1253,4 @@
   "workflowAnalytics.averageTaskDurationPerUser.user": "User",
   "workflowAnalytics.averageWorkflowDuration.heading": "Average Workflow Duration",
   "workflowAnalytics.workflowInitiators.heading": "Workflow Initiators"
-}
```
Comment (RSalem07) [inline i18n/messages.json:1253]: I am wondering where these new labels are used :). Perhaps I missed something

Comment (gcarlin) [reply i18n/messages.json:1253]: Probably on the Analytics, no issue  there

**Comment by gcarlin** [comment]
I've amended the commit. All should be fine now

File: i18n/messages-es-ES.json [hunk 2]
```diff
@@ -1250,4 +1253,4 @@
   "workflowAnalytics.averageTaskDurationPerUser.user": "Usuario",
   "workflowAnalytics.averageWorkflowDuration.heading": "Duración promedio del flujo de trabajo",
   "workflowAnalytics.workflowInitiators.heading": "Iniciadores de flujo de trabajo"
-}
\ No newline at end of file
+}
```
Comment (nmpcunha) [inline i18n/messages-es-ES.json:1256]: Still adding characters here.
Probably some configuration from the IDE?

Comment (gcarlin) [reply i18n/messages-es-ES.json:1256]: Yes exactly

Comment (gcarlin) [reply i18n/messages-es-ES.json]: Should be good now :)

**Comment by gcarlin** [comment]
@nmpcunha Git commits are now clean:)

File: i18n/messages.json [hunk 3]
```diff
@@ -140,6 +140,9 @@
   "app.title.favorites": "Favorites",
   "app.title.home": "Home",
   "app.title.profile": "Profile",
+  "app.title.retentionEvents" : "Retention Events",
+  "app.title.retentionRules" : "Retention Rules",
+  "app.title.retentionSearch" : "Retention Search",
   "app.title.search": "Search",
   "app.title.search.assets": "Assets",
   "app.title.search.default": "Default",
```
Comment (RSalem07) [inline i18n/messages.json:148]: no space between the key and `:`:

`"app.title.retentionEvents" : "Retention Events",` -> `"app.title.retentionEvents": "Retention Events",`

Comment (RSalem07) [reply i18n/messages.json:148]: same for the three modifications

Comment (gcarlin) [reply i18n/messages.json:148]: fixed

**Pull request merged**